mk/submodule.mk

```diff
@@ -0,0 +1,9 @@
+# libtuv related variables
+LIBTUV_PATH = deps/libtuv
+LIBTUV_INCLUDE_PATH := $(LIBTUV_PATH)/include
```

Comment:
Change from `LIBTUV_INCLUDE_PATH` to `LIBTUV_INCLUDE` and re-assign it to `-I $(LIBTUV_PATH)/include`.

Reply:
Alternatively, we can follow the naming convention of [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config/) to rename libtuv-specific variables as `LIBTUV_CFLAGS` and `LIBTUV_LIBS`.

Reply:
I am curious about the benefit of this writing skill.

Reply:
No visible benefit but consistency. The reason why `LIBTUV_INCLUDE` or `LIBTUV_CFLAGS` should contain `-I` prefix is to reorder search paths if necessary.

Reply:
Done.